.github/playground-wordpress-net---issues.md

```diff
@@ -0,0 +1,36 @@
+---
+name: playground.wordpress.net - Issues
+about: Something did not work as expected in the online version of Playground?
+title: 'Playground Online: Report an issue with playground.wordpress.net'
```

Comment:
```suggestion
title: 'Bug Report – playground.wordpress.net'
```

Btw how is `name` different than `title`? Also, shouldn't this be a `.yml` file living in `.github/ISSUE_TEMPLATE`? Or will it work as a `.md` file living in `.github`?

Reply:
> Btw how is name different than title? 

- `title`: the title of the issue that will be opened. The person submitting it should change the title, but it's a bit like the `placeholder` attribute in HTML `<input>` elements.
- `name`: the name of the template (required).

***

> shouldn't this be a .yml file living in .github/ISSUE_TEMPLATE? 

[It can be either](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository) `.md` or `.yml`. Markdown is easier to style :-) You can see [an example in Learn's repo](https://github.com/WordPress/Learn/tree/trunk/.github/ISSUE_TEMPLATE).
If we want [template forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms), we'd need `.yml`